envoy/config/rbac/v2alpha/local_rbac.proto

```diff
@@ -0,0 +1,63 @@
+syntax = "proto3";
+
+package envoy.config.rbac.v2alpha;
+
+import "envoy/config/rbac/v2alpha/rbac.proto";
+
+// LocalRBAC specifies RBAC policies that apply to the local RBAC engine
+// that typically runs on a proxy. There can be multiple services running behind
+// a proxy.
+//
+// Here is an example of local RBAC policy, which contains policies for a single service.
+// {
+//   "products.default.svc.cluster.local": {
```

Comment:
Can you use pure YAML in your examples without superfluous JS braces? I think this will make it a bit easier to read.

Reply:
Will do. Thanks.

Reply:
Done. I moved this example to README.md.